### PR TITLE
Fix/observatory cors

### PR DESCRIPTION
**Pull request by jonathanfallon**

### CODE REVIEW

File: api/proxy/src/HttpTransport.ts [hunk 1]
```diff
@@ -172,21 +172,30 @@ export class HttpTransport implements TransportInterface {
...
       }),
     );
+
+    // Use CORS asynchronously to log the calls and check against a list of domains
     this.app.use(
-      /\/observatory.*|geo\/search/i,
-      cors({
-        origin: '*',
-        optionsSuccessStatus: 200,
+      '/observatory',
```
Comment (nmrgt) [inline api/proxy/src/HttpTransport.ts:186]: est-ce que ça inclue les sous-routes ? Genre /observatory/toto ?

File: api/proxy/src/HttpTransport.ts [hunk 2]
```diff
@@ -172,21 +172,30 @@ export class HttpTransport implements TransportInterface {
...
+      cors((req: Request, callback) => {
+        const domains = [
+          'https://demo.covoiturage.beta.gouv.fr',
+          'https://covoiturage.beta.gouv.fr',
+          'http://localhost:4200',
```
Comment (nmrgt) [inline api/proxy/src/HttpTransport.ts:191]: pourquoi ne pas utiliser APP_SHOWCASE_URL ?

**Comment by jonathanfallon** [comment]
# ☁️ Nx Cloud Report

CI is running/has finished running commands for commit 669724399bac4239332a62de8298abe236967451. As they complete they will appear below. Click to see the status, the terminal output, and the build insights.

[📂 See all runs for this branch](https://cloud.nx.app/branch?workspaceId=63ebd539a53ecd331737f524&branchName=2259,fix/observatory-cors)

---


<details>
<summary>✅ Successfully ran 1 target </summary>

- [`lerna run build`](https://cloud.nx.app/runs/66tMz4Ni9m)
</details>

---

Sent with 💌 from [NxCloud.](https://nx.app)

File: api/proxy/src/HttpTransport.ts (hunk 1 above)
Comment (jonathanfallon) [reply api/proxy/src/HttpTransport.ts:186]: Oui ça fonctionne pour `/observatory/monthly-flux/last`

**Pull request merged**